_data/projects/asp.net-javascript-services.yml

```diff
@@ -0,0 +1,18 @@
+name: ASP.NET Javascript Services
+desc: >
+  ASP.NET Javascript Services is a set of technologies for ASP.NET Core developers. 
+  It provides infrastructure that you'll find useful if you use Angular 2 / React / Knockout / etc. on the client, 
+  or if you build your client-side resources using Webpack, 
+  or otherwise want to execute JavaScript on the server at runtime.
+site: https://github.com/aspnet/javascriptservices
+tags:
```

Comment:
great idea... I added a few. Let me know if you think of anymore that should be added before the pr gets merged in.